examples/inline-yaml.cf

```diff
@@ -0,0 +1,49 @@
+#+begin_src cfengine3
+bundle agent example_inline_yaml
+#@ brief Example illustrating inline yaml
+{
+  vars:
+      # YAML requires "---" header (followed by newline)
+      # NOTE \n is not interpreted as a newline, instead use $(const.n)
+
+      "yaml_multi_line" data => '---
+- key1: value1
+- key2: value2';
+
+
+      "yaml_single_line" data => '---$(const.n)- key1: value1$(const.n)- key2: value2';
```

Comment:
Totally looking for something to criticize in this beautiful example: maybe use a different key and value pattern in the second container?

Reply:
Aaaaaaaaaas yoooooooooooooooooooooooooou wiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiish